pkg/network/protocols/http/model_linux.go

```diff
@@ -123,8 +123,8 @@ func (tx *EbpfHttpTx) String() string {
 // 10 bits precision (any value will be +/- 1/1024)
 const roundMask uint64 = 1 << 10
 
-// nsTimestampToFloat converts a nanosec timestamp into a float nanosecond timestamp truncated to a fixed precision
-func nsTimestampToFloat(ns uint64) float64 {
+// NSTimestampToFloat converts a nanosec timestamp into a float nanosecond timestamp truncated to a fixed precision
+func NSTimestampToFloat(ns uint64) float64 {
```

Comment:
should it be exported?

Reply:
yes, it is being used in http2 as well `http2/model_linux.go:46`

Reply:
so can you move it to common? if it is shard thro the http and http2

Reply:
moved